lib/fog/vcloud_director/models/compute/disks.rb

```diff
@@ -32,6 +32,11 @@ def item_list
           end
           items
         end
+        
+        # Filters out all the controllers (returns only resource_type == 17)
```

Comment:
Can you be more explicit here and explain what a disk of `resource_type` equals 17 is?